lib/sorcery/model/temporary_token.rb

```diff
@@ -12,7 +12,7 @@ def self.included(base)
 
       # Random code, used for salt and temporary tokens.
       def self.generate_random_token
-        SecureRandom.urlsafe_base64(15).tr('lIO0', 'sxyz')
+        SecureRandom.urlsafe_base64(Controller::Config.token_randomness).tr('lIO0', 'sxyz')
```

Comment:
> [15, Controller::Config.token_randomness].max

It is weird that it isn't set the specified value by user.  So, I think the specified value should always be set.